test/dynamodb/dynamodb.js

```diff
@@ -0,0 +1,100 @@
+const ddblocal = require('local-dynamo'),
+      aws = require('./helper.js'),
+      mockItems = require('../../mock/feeds.json');
```

Comment:
could you move the mock and expectation JSON files into `/test/fixtures` so that it is inline with our other repos?